apps/api/src/app/channeling/channeling.service.ts

```diff
@@ -1,14 +1,19 @@
 import { Injectable } from '@nestjs/common';
 import { Channeling } from './interfaces/channeling.interface';
+import { ChannelingSession } from './interfaces/channelingSession.interface';
```

Comment:
file imports must be placed after dependency imports

Reply:
changed